### DISCUSSION TITLE
Is it possible to build a model using my own dataset?

### DISCUSSION THREAD

**Discussion by VladHornai** [NVIDIA-NeMo/NeMo#5199] [Q&A]
Hello,
I a newbie with respect to using NeMo. Could somebody give me some hints regarding starting to develop an ASR model using my own dataset. I am at the beginning of parsing the documentation. I would appreciate very much some ideas that guide me further.
Thanks !

**Comment by titu1994** [comment]
There's quite a few tutorials for ASR, I'd suggest going through them but overall yes, as long as you can setup a manifest file, you can easily train from scratch or fine-tune a pre-existing model on your own dataset
